js/modules/loki_public_chat_api.js

```diff
@@ -561,6 +580,17 @@ class LokiPublicChannelAPI {
           if (!from) {
             ({ from } = noteValue);
           }
+
+          // check for markdown
+          if (markdown) {
+            // DO NOT ALLOW users/servers to slip in their own custom HTML
+            const file = mdEngine.processSync(markdown);
+            text = String(file);
```

Comment:
Is there any way we can do the markdown processing at a lower level? There seems to be some duplicated logic between public and private messages.
Why not emit the raw message here and leave the markdown handling to the message.js model?

Reply:
Mainly to keep it compatible with the appdotnet ecosystem (like the irc bridges, etc). This requires no markdown in the `text` field. The duplicate logic is contained to the libloki_markdown library itself.